src/main/java/com/derongan/minecraft/deeperworld/listeners/MovementListener.kt

```diff
@@ -29,6 +39,16 @@ object MovementListener : Listener {
         }
     }
 
+    @EventHandler(ignoreCancelled = true, priority = EventPriority.LOW)
+    fun onVehicleMove(vehicleMoveEvent: VehicleMoveEvent) {
+        val (vehicle) = vehicleMoveEvent
+        val players = vehicle.getPlayerPassengersRecursive() ?: return
```

Comment:
If you decide to go with the empty list instead and wanted to return right here if empty, I think I'd put this on the next line:
```kotlin
if (players.isEmpty()) return
```
But the neat thing with an empty list is that `firstOrNull` will just return null and the case is handled for us, so we don't even need to do that.